drizzlepac/hlautils/astrometric_utils.py

```diff
@@ -189,7 +165,7 @@ def create_astrometric_catalog(inputs, **pars):
     for source in ref_dict:
         if 'GAIAsourceID' in source:
             g = source['GAIAsourceID']
-            if gaia_only and g.strip() is '':
+            if gaia_only and g.strip() == '':
```

Comment:
Several issues here:
1. [minor,style]: `if gaia_only and not g.strip()'':` (empty strings, lists evaluate to `False`)
2. [inconsistency]: `g.strip()` implies that `g` is a string while `g = -1` below makes it an integer. Then in the line below, `ref_table.add_row((r, d, m, o, g))` what is the column type for `g`? Maybe, instead of assigning a negative integer, assign a string: `g = '-1'`?